.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "BioUtils"]
 	path = BioUtils
-	url = git@github.com:allista/BioUtils.git
+	url = https://github.com/allista/BioUtils.git
```

Comment:
This is not necessary. GitHub is smart enough to recognize public repo in submodules and clone it through https. Just checked out with a fresh user.